backend/src/middleware/languages/languages.js

```diff
@@ -0,0 +1,42 @@
+import LanguageDetect from 'languagedetect'
+import sanitizeHtml from 'sanitize-html'
+
+const removeHtmlTags = (input) => {
+  return sanitizeHtml(input, {
+    allowedTags: [],
+    allowedAttributes: {},
+  })
+}
+
+const setPostLanguage = (text) => {
+  const lngDetector = new LanguageDetect()
+  lngDetector.setLanguageType('iso2')
+  const result = lngDetector.detect(removeHtmlTags(text), 2)
```

Comment:
Will this always return two results? Because else the lines below could fail, since `result` might not contain a first and second element.